src/components/bridge/Bridge.vue

```diff
@@ -173,11 +174,11 @@
         var tokenType
         var erc20;
         //gets bech32 user address
-        if(this.getBridgeMode() == EXCHANGE_MODE.ONE_TO_ETH){
```

Comment:
Maybe I am missing something but it seemed like both blocks of this `if/else` did the same thing, so I consolidated it.